geo.txt

```diff
@@ -56,3 +56,1703 @@ Phys 91(11):957-962
+CH3 group. J Comput Chem 35(32):2333-2342
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+B H   1.1900(1)
+B F   1.3200(3)
+B O   1.3464(3)
+O H   0.9584(3)
+H B F 119.38(10)
+F B O 117.203(5)
+B O H 122.65(3)
+
+NAME = BF2
+# Point Group: C2v
+# Shape: Bent
+# Bond lengths: r0     (angstroms)
+# Bond angles:  theta0 (degrees)
+# Source Paper: Yang J, Ellis B, Clouthier DJ (2011) The complex spectrum of a “simple” free radical: The A-X band system of
+the jet-cooled boron difluoride free radical. J Chem Phys 135(9):094305/1-094305/9
+doi:10.1063/1.3624528
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+B F   1.3102(9)
+F B F 119.7(6)
+
+NAME = BF2HO
+# Point Group: Cs
+# Shape: Planar
+# Bond lengths: re(se)     (angstroms)
+# Bond angles:  thetae(se) (degrees)
+# Source Paper: Vogt N, Demaison J, Rudolph HD, Perrin A (2015) Interplay of experiment and theory: high resolution infrared
+spectrum and accurate equilibrium structure of BF2OH. Phys Chem Chem Phys 17(45):30440-30449
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+B F(1)   1.3239(2)
+B F(2)   1.3139(2)
+B O      1.3426(3)
+O H      0.9581(6)
+O B F(1) 122.31(2)
+B O H    113.09(4)
+O B F(2) 119.47(2)
+F B F    118.22(3)
+
+NAME = BH2
+# Point Group: C2v
+# Shape: Bent
+# Bond lengths: r0     (angstroms)
+# Bond angles:  theta0 (degrees)
+# Source Paper: Sunahori FX, Gharaibeh M, Clouthier DJ, Tarroni R (2015) BH2 revisited: New, extensive measurements of
+laser-induced fluorescence transitions and ab initio calculations of near-spectroscopic accuracy. J Chem Phys
+142(17):174302/1-174302/13
+[http://dx.doi.org/10.1063/1.4919094]
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+B H   1.197(2)
+H B H 129.6(2)
+
+NAME = BH6N
+# Point Group: C3v
+# Shape: Trigonal pyramidal --- Trigonal pyramidal
+# Bond lengths: rm^(1)     (angstroms)
+# Bond angles:  thetam^(1) (degrees)
+# Source Paper: Demaison J, Lievin J, Császár AG, Gutle C (2008) Equilibrium structure and torsional barrier of BH3NH3. J
+Phys Chem A 112(19):4477-4482
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+B...N   1.6576(16)
+B   H   1.2098(2)
+N   H   1.0083(3)
+N...B H 104.73(1)
+B...N H 110.29(2)
+
+# Bond lengths: re^se     (angstroms)
+# Bond angles:  thetae^se (degrees)
+
+B...N   1.6453(1)
+B   H   1.2058(3)
+N   H   1.0101(3)
+N...B H 105.00(2)
+B...N H 110.97(2)
+
+NAME = BeI2
+# Point Group: Dooh
+# Shape: Bent
+# Bond lengths: rg     (angstroms)
+# Bond angles:  thetag (degrees)
+# Source Paper: Shlykov SA, Zhabanov YA, Giricheva NI, Girichev AG, Girichev GV (2015) Combined gas electron
+diffraction/mass spectrometric study of beryllium diiodide assisted by quantum chemical calculations: structure
+and thermodynamics of beryllium dihalides. Struct Chem 26 (5-6):1451-1458
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+# Temperature = 501(5) Kelvin
+
+Be I   2.163(6)
+I  Be I 165(3)
+
+# Temperature = 722(10) Kelvin
+
+Be I   2.172(7)
+I  Be I 162(4)
+
```

Comment:
I wonder why you included BeI2, but not Be2O?

Reply:
I see. It's the whole molecule that's an ion, so I would just have treated it just like all of the other molecules (in terms of the way that you add all of the data), then I would have just added a comment with the charge of the molecule. Now you know for next time, to please include ions when adding molecules. But you don't need to add this one because it's a triatomic.